.github/workflows/ember.yml

```diff
@@ -0,0 +1,370 @@
+name: "Ember"
+
+on:
+  workflow_call:
+    inputs:
+      DEPLOYMENT:
+        description: "The name of the Kubernetes deployment to deploy to, ex: app-app-deployment"
+        type: string
+        required: true
+      CONTAINER:
+        description: "The name of the container within the Kubernetes deployment to patch, ex: app-app-deployment -c app-app"
+        type: string
+        required: true
+
+env:
+  CONTAINER_REGISTRY: linced.azurecr.io/linc-ed
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    outputs:
+      image_tag_build: ${{ steps.tag.outputs.image_tag_build }}
+      image_tag_branch: ${{ steps.tag.outputs.image_tag_branch }}
+    steps:
+
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Setup Node 10.21"
+        uses: actions/setup-node@v3
+        with:
+          node-version: 10.21
+          cache: 'npm'
+          cache-dependency-path: linced-*/package-lock.json
+
+      - name: "BitBucket Keyscan"
+        id: scan
+        run: echo "::set-output name=known_hosts::$(ssh-keyscan -t rsa bitbucket.org)"
+
+      - name: "BitBucket SSH Key Install"
```

Comment:
Remove me :)

Reply:
All else resolved, will leave this to another day as it might be the only thing allowing us to pull GitHub repos :polite-cat:

Reply:
So yeah that key is required for us to auth with Github currently, I have since removed it from my BitBucket account which is fine. Should try to use HTTPS for the default CI token / rename the step to Github SSH